tests/test_spark.py

```diff
@@ -187,17 +198,18 @@ def _get_hdfs_options():
     if _is_strict():
         options = {'service': {'principal': 'service-acct', 'secret_name': 'secret'}}
     else:
-        options = {}
+        options = {"service": {}}
+
+    options["service"]["beta-optin"] = True
```

Comment:
The beta-hdfs changes in this file are just to get the tests working with the latest breaking hdfs change.